### ISSUE TITLE
Cannot open .py files from Box

### ISSUE THREAD

**Issue by nicksnz** [spyder-ide/spyder#11985]
### Issue Report Checklist

* [y ] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [y ] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [y ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ y] Tried basic troubleshooting (if a bug/error)
    * [ y] Restarted Spyder
    * [ y] Reset preferences with ``spyder --reset``
    * [ y] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ y] Tried the other applicable steps from the Troubleshooting Guide
* [ y] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
Cannot open python files stored in Box. If I click open file then nothing happens. In the right hand files pane, the files that I cannot open appear as below, the test.py file in the same folder opens and runs correctly.

If I right-click on the .py files and run from the files pane, then it opens a new console window and runs the file from there (but I would like to be able to edit them).

The files that do not run are stored in a Box cloud folder, and I copied them into Box. The test.py (which opens fine) I created in Box.

![file_not_open](https://user-images.githubusercontent.com/62315928/77593392-fe271200-6f58-11ea-8b28-6f2eec1b1f2b.PNG)

### What steps reproduce the problem?

1. Copy .py file into Box
2. Open .py file from spyder

### What is the expected output? What do you see instead?

Expecting .py files to open in editor.

### Paste Traceback/Error Below (if applicable)

```python-traceback

PASTE TRACEBACK HERE

```

## Versions

* Spyder version: 4.1.1
* Python version: 3.7.7
* Qt version: 5.9.6
* PyQt version: 5.9.2
* Operating System name/version: Windows 10

### Dependencies

# Mandatory:
atomicwrites >=1.2.0           :  1.3.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.3.0 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.4.4 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.9 (OK)
qdarkstyle >=2.8               :  2.8 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.1 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.3 (OK)
sphinx >=0.6.6                 :  2.4.0 (OK)
spyder_kernels >=1.9.0;<1.10.0 :  1.9.0 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  0.29.15 (OK)
matplotlib >=2.0.0             :  3.1.3 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  1.0.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  1.5.1 (OK)

**Comment by ccordoba12** [comment]
Please upload one of your `Q2*_importance_*.py` files so we can take a look at them. The problem is that they are detected as binary, but without seeing their contents, we won't be able to understand why.

**Comment by goanpeca** [comment]
Could it be that there is synchronization on demand (like dropbox) so that files in the computer are "dummy" files  and only when clicking them (from the system fileexplorer) they get downloaded?

**Comment by ccordoba12** [comment]
Interesting @goanpeca. That could certainly be the problem here.

**Comment by nicksnz** [comment]
Thanks @ccordoba12 @goanpeca 

Attached is an example file (saved as txt). Interestingly when I opened the .py file in notepad and then resaved into the same Box folder spyder could then open it correctly.

I think the issue is something like what @goanpeca said. I did try the option to 'Make files available offline' in Box but that didn't work. Thanks

[Q21_importance_discounts_EXAMPLETOUPLOAD.txt](https://github.com/spyder-ide/spyder/files/4391749/Q21_importance_discounts_EXAMPLETOUPLOAD.txt)
